src/modules/connections/connections.controller.ts

```diff
@@ -47,4 +48,12 @@ export class ConnectionsController {
   getDescription(@Param() { id }: ConnectionTablesDto) {
     return this.connectionsService.getConnectionTables(id);
   }
+
+  @ApiBearerAuth()
+  @ApiOkResponse({type: []})
+  // @UseGuards(AuthGuard('user'))
+  @Get('check/:id')
```

Comment:
Maybe change to :id/check as we have already :id/tables

Reply:
ko